test/test_relational_sklearn.py

```diff
@@ -0,0 +1,177 @@
+# Copyright 2019 IBM Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+from sklearn.preprocessing import MinMaxScaler as SkMinMaxScaler
+
+from lale.datasets.multitable.fetch_datasets import fetch_go_sales_dataset
+from lale.lib.rasl import MinMaxScaler as RaslMinMaxScaler
+
+
+class TestMinMaxScaler(unittest.TestCase):
+    def setUp(self):
+        self.go_sales = fetch_go_sales_dataset()
+
+    def test_fit(self):
+        columns = ["Product number", "Quantity", "Retailer code"]
+        data = self.go_sales[0][columns]
+        sk_scaler = SkMinMaxScaler()
+        rasl_scaler = RaslMinMaxScaler()
+        sk_trainned = sk_scaler.fit(data)
+        rasl_trainned = rasl_scaler.fit(data)
+        self.assertTrue((sk_trainned.data_min_ == rasl_trainned.impl.data_min_).all())
```

Comment:
How about this instead?
```python
self.assertEqual(list(sk_trainned.data_min_), list(rasl_trainned.impl.data_min_))
```

Reply:
What is the benefit of this approach?

Reply:
The benefit is better diagnostic messages: if the assertion fails, it prints the two values being compared.

Reply:
See commit 7d1e2f1